Contributing docs: explain `compat_features` versus tags (#1667)
web-platform-dx/web-features: docs/CONTRIBUTING.md

| `spec` | One or more specification URLs for this feature. | String, or array of strings | Yes |
| `group` | An optional group, or list of groups that this feature belongs to. See the definition of groups under [Create a new feature from scratch](#create-a-new-feature-from-scratch). | String, or array of strings | No |
| `caniuse` | The feature's ID on the [Can I Use](https://caniuse.com/) website. | String | No |

> | `compat_features` | @mdn/browser-compat-data (BCD) entry keys (e.g., `css.properties.background-color`) that make up this feature. If there is no `compat_features` key and there exists a matching `web-features:<feature-id>` tag in BCD, then this key is automatically populated from the tag. | Array of strings | No |

Reviewer: ```suggestion
| `compat_features` | @mdn/browser-compat-data (BCD) entry keys (e.g., `css.properties.background-color`) that make up this feature. If `compat_features` is not set in `<feature-id>.yml`, the `dist` script will populate `compat_features` in `<feature-id>.yml.dist` with BCD entry keys tagged with `web-features:<feature-id>` in BCD, if any exist. | Array of strings | No |
```

I tried to clarify the distinction between the `yml` and `yml.dist` file here, but I'll defer to you on if this is a helpful distinction to make here.

Author: Good idea. I like making this more concrete. 👍
